join/join-testnet-with-binaries.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
In the other scripts I moved this instruction to immediately before launching the node, just so that we don't have the script halt over a `grep` that didn't select anything. Given the amount of checks we do in the beginning, I think its fine